src/Azs.InfrastructureInsights.Admin/test/common.ps1

```diff
@@ -1,4 +1,8 @@
-﻿function ValidateMetrics{
+﻿$global:SkippedTests = @("TestCloseAlert")
+$global:Location = "redmond"
```

Comment:
If location & RgName are added to default parameters, its not required to hardcode here as a global variable. You can simply remove the location parameter and resource group name parameter from any cmdlet.

Reply:
Removed.